chunked_stream/pubspec.yaml

```diff
@@ -9,5 +9,6 @@ issue_tracker: https://github.com/google/dart-neats/labels/pkg:chunked_stream
 dev_dependencies:
   test: ^1.16.0-nullsafety
   pedantic: ^1.4.0
+  meta: ^1.3.0-nullsafety.6
```

Comment:
Considering that we use `@sealed` in `lib/`, this should probably be a regular dependency?

Reply:
absolutely, that was a mistake.. Good catch.